cgmanifests/cgmanifest.json

```diff
@@ -23,16 +23,7 @@
           "maven": { "GroupId": "com.google.protobuf", "ArtifactId": "protobuf-java", "Version": "3.20.1" },
           "DevelopmentDependency": true
         }
-      },
-      {
-         "component": {
-            "type": "git",
-            "git": {
-               "commitHash": "a3534567187d2edc428efd3f13466ff75fe5805c",
-               "repositoryUrl": "https://github.com/microsoft/gsl.git"
-            }
-         }
-      },
+      },      
       {
          "component": {
             "type": "git",
```

Comment:
Thank you for pointing it out. It was my oversight. 
I want to discuss it with you: right now our nuget packaging pipeline is failing because its build environment does not have git. So I suggest changing the download method of GSL from git to https. Then I can add GSL to the deps.txt. Then when I re-run the python script in this directory, it will be added.

Reply:
> So I suggest changing the download method of GSL from git to https.

that's fine with me